packages/create-tina-app/src/index.ts

```diff
@@ -2,27 +2,36 @@ import { Telemetry } from '@tinacms/metrics'
 import { Command } from 'commander'
 import prompts from 'prompts'
 import path from 'node:path'
-//@ts-ignore
 import { version, name } from '../package.json'
 import { isWriteable, makeDir, isFolderEmpty } from './util/fileUtil'
 import { install } from './util/install'
 import chalk from 'chalk'
 import { tryGitInit } from './util/git'
-import { exit } from 'node:process'
-import { EXAMPLES, downloadExample } from './examples'
+import { TEMPLATES, downloadTemplate } from './templates'
 import { preRunChecks } from './util/preRunChecks'
 
+const PKG_MANAGERS = ['yarn', 'npm', 'pnpm']
```

Comment:
```suggestion
const PKG_MANAGERS = ['npm', 'yarn', 'pnpm']
```
best to have npm first since we know that they'll have this installed (otherwise they wouldn't have been able to run `npx`)